tests/dummy/app/controllers/chips.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  fruitNames: Ember.A(['Apple', 'Banana', 'Orange']),
+
+  customFruitNames: Ember.A(['Apple', 'Banana', 'Orange']),
+
+  contacts: Ember.A([{
+    name: 'James',
+    email: 'james@ember.com',
+    image: 'tomster.png'
+  }]),
+
+  allContacts: Ember.A([{
```

Comment:
Can we display these in the demo app so the user can use them better? Also, the demo raises an assertion regarding email.